i18n/fr.jsonp.js

```diff
@@ -19,8 +19,14 @@ document.localeJson = {
       "configure-title": "Configuration",
       "configure-about-usage-link": "Informations (Utilisation)",
       "configure-btn-library": "Bibliothèque ZIM",
+      "configure-btn-folderselect": "Sélectionner un dossier",
+      "configure-btn-rescan": "Rechercher",
+      "configure-about-rescan-tip": "Réanalyser la carte SD et la mérmoire interne",
+      "configure-select-file-numbers": "archive(s) trouvées dans le lieu sélectionné. ",
       "configure-download-instructions": "Cette application a besoin d'un fichier ZIM pour fonctionner.<br />Pour des instructions complètes, veuillez consulter la section",
       "configure-select-instructions": "Veuillez sélectionner ou glisser-déposer un fichier .zim (ou tous les .zimaa, .zimab etc. dans le cas d'un fichier ZIM découpé)&nbsp;:",
+      "configure-select-file-instructions": "Veuillez sélectionner l'archive que vous souhaitez utiliser :",
```

Comment:
Space before colon **is** correct here.

Reply:
I am not sure how to do it. In the code we are using `.innerText` property to change text `&nbsp` will just show as text